includes/admin/class-coblocks-url-generator.php

```diff
@@ -44,7 +44,7 @@ public function get_store_url( $path = '', $params = array() ) {
 
 		$params = array_merge( $params, $id );
 
-		$url = trailingslashit( COBLOCKS_SHOP_URL . $path ) . '?' . http_build_query( $params, '', '&#038;' );
+		$url = trailingslashit( 'https://coblocks.com/' . $path ) . '?' . http_build_query( $params, '', '&#038;' );
```

Comment:
@paranoia1906 I'd probably use [add_query_arg()](https://developer.wordpress.org/reference/functions/add_query_arg/) here.

Reply:
Using the built-in `add_query_arg()` makes sense to me. Commit is updated to include this change. Here an example of output URL. 

New: 
```
https://coblocks.com/?utm_medium=coblocks-lite&utm_source=plugins-page&utm_campaign=plugins-row&utm_content=help-and-faqs
```
Old: 
```
https://coblocks.com/?utm_medium=coblocks-lite&utm_source=plugins-page&utm_campaign=plugins-row&utm_content=help-and-faqs
```